pkg/vm/engine/tae/catalog/basemvccnode.go

```diff
@@ -199,7 +203,8 @@ type BaseNode[T any] interface {
 type MVCCNode[T BaseNode[T]] struct {
 	*EntryMVCCNode
```

Comment:
```go
type MVCCNode[T BaseNode[T]] struct {
    EntryMVCCNode
    txnbase.TxnMVCCNode
    BaseNode         T
    CommitSideEffect func(commitTs types.TS) // used for object replay, no need to persist
}
```
We can change the definition as above to reduce the number of small objects

Reply:
done